pyiron/vasp/base.py

```diff
@@ -1341,6 +1341,15 @@ def get_nelect(self):
         else:
             return self["output/generic/dft/n_elect"]
 
+    def get_magnetic_moments(self):
+        """
+        Gives the magnetic moments of a calculation
```

Comment:
Maybe we need to mention they are given for each snapshot.

Reply:
Fair enough. I changed the docstrings.